porch/controllers/packagevariantsets/api/v1alpha2/packagevariantset_conversion.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 The kpt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha2
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+
+	pkgvarapi "github.com/GoogleContainerTools/kpt/porch/controllers/packagevariants/api/v1alpha1"
+	v1alpha1 "github.com/GoogleContainerTools/kpt/porch/controllers/packagevariantsets/api/v1alpha1"
+)
+
+func (src *PackageVariantSet) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v1alpha1.PackageVariantSet)
```

Comment:
Is my understanding correct that we won't be using this code until we add the conversion webhooks? If that's the case (and if we are not adding the conversion webhooks in this PR), might it make more sense to leave these files out of this PR, and add this code back in with the conversion webhooks?

Reply:
Yes, that's correct. I could do that.

Reply:
Done.